### PR TITLE
New Fairs rail

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Working Fairs rail
- [x] Tests

Please note the ordering of rails is expected to change as we near the finish of the Home/ForYou scene; I put Fairs at the top for now.

simulator screen shot nov 6 2017 1 03 47 pm

Fixes https://github.com/artsy/collector-experience/issues/454 

Skip New Tests

### CODE REVIEW

**Comment by katarinabatina** [comment]
@sarahscott 👌  looks awesome! You mind @'ing me when you've merged?

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by sarahscott** [comment]
@katarinabatina it's up for code review now 🌺

File: src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import * as React from "react"
```
Comment (l2succes) [inline src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx]: we should use the following to import react from now on.
`import React from "react"`

I explain the reasoning in this PR (https://github.com/artsy/emission/pull/820)

File: src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+const IconCarousel = styled.ScrollView`
+  flex-direction: row;
+  margin-top: 10;
+`
+
+const IconTapArea = styled.TouchableHighlight`
+  height: 80;
+  width: 80;
+  border-radius: 40;
+  marginRight: 7;
```
Comment (l2succes) [inline src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx]: nitpicking: `margin-right`

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by sarahscott** [comment]
@l2succes just realized matt is OOO - reassigning to you!

**Pull request merged**